eZ/Publish/Core/MVC/Symfony/Controller/Content/DownloadController.php

```diff
@@ -52,14 +47,11 @@ public function downloadBinaryFileAction( $contentId, $fieldIdentifier, $filenam
             $content = $this->contentService->loadContent( $contentId );
         }
 
-        $field = $this->translationHelper->getTranslatedField(
-            $content, $fieldIdentifier,
-            $request->query->has( 'language' ) ? $request->query->get( 'language' ) : null
-        );
-        if ( !$field instanceof Field )
+        $field = $this->findField( $content, $fieldId );
```

Comment:
but if not using translating helper, how can this now which one to download? or do we have distinct fieldId per every version of the attribute?